src/provider/ChannelModel.ts

```diff
@@ -90,14 +90,17 @@ export class ChannelModel {
         const previousChannelId = this._identityHashToChannelIdMap.get(identityHash) || GLOBAL_CHANNEL_ID;
```

Comment:
If we're concerned about a bad identity passed here, should we have an `identity.name !== undefined` check here?

Reply:
For the purposes of the prototype, I'm not concerned about bad identities, and that's probably a thing to check outside of this anyway

Reply:
The sanitizing is more just so that the data we send out is 'clean'. It was primarily motivated to make test-writing easier, but it feels like the the 'right' thing to do anyway

Reply:
Yeah, we'll eventually do the sanitising at the client/APIHandler level, rather than having to put that kind of thing all over the provider.

Doing it here is good for the time being, though.